source/features/default-branch-button.tsx

```diff
@@ -32,38 +32,37 @@ const getUrl = memoize(async (currentUrl: string): Promise<string> => {
 });
 
 async function add(branchSelector: HTMLElement): Promise<void> {
-	// React issues. Duplicates appear after a color scheme update
-	// https://github.com/refined-github/refined-github/issues/7098
-	if (elementExists('.rgh-default-branch-button')) {
-		return;
-	}
```

Comment:
I'd rather keep this comment *and* this early return. 

To fix the grouping issue, you can copy-paste the groupButtons call here (or make a function)